test/pkg/debug/events.go

```diff
@@ -52,7 +52,7 @@ func (c *EventClient) DumpEvents(ctx context.Context) error {
 
 func (c *EventClient) dumpKarpenterEvents(ctx context.Context) error {
 	el := &v1.EventList{}
-	if err := c.kubeClient.List(ctx, el, client.InNamespace("karpenter")); err != nil {
+	if err := c.kubeClient.List(ctx, el, client.InNamespace("kube-system")); err != nil {
```

Comment:
Feels like this should be a constant somewhere rather than us typing it in manually